crates/task-impls/src/network.rs

```diff
@@ -85,6 +86,9 @@ impl<TYPES: NodeType> NetworkMessageTaskState<TYPES> {
                             GeneralConsensusMessage::TimeoutVote(message) => {
                                 HotShotEvent::TimeoutVoteRecv(message)
                             }
+                            GeneralConsensusMessage::UpgradeCertificate(message) => {
+                                HotShotEvent::UpgradeCertificateRecv(message)
+                            }
```

Comment:
Related to the comment in events.rs, should we also handle the other vote/cert events here?

Reply:
Yes - added!